<?xml version="1.0"?>
<launch>
<arg name="n_robots" default="3" />
<!-- <arg name="deg_type" default="wdeg" /> -->
<arg name="human_pos" default="['(0,0)']" />
<arg name="radius" default="10" />
<arg name="env" default="room_lvl7" />
<arg name="update_rate_info" default="1.0" />

<include file="$(find best_vertices)/launch/map_server.launch">
  <arg name="env" value="$(arg env)" />
</include>

<include file="$(find best_vertices)/launch/graph_generator.launch">
  <arg name="cfg" value="$(arg env)" />
</include>

<node pkg="best_vertices" type="best_vertices_node.py" name="best_vertices_node" output="screen">
  <param name="n_robots" value="$(arg n_robots)" />
  <!-- <param name="deg_type" value="$(arg deg_type)" /> -->
  <rosparam param="human_pos" subst_value="True">$(arg human_pos)</rosparam> 
  <param name="radius" value="$(arg radius)" />
  <param name="env" value="$(arg env)" />
</node>

<include file="$(find best_vertices)/launch/rviz.launch">
  <arg name="cfg" value="$(arg env)"/> 
</include>

</launch>